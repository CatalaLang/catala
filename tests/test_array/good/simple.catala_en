## Article

```catala
declaration scope A:
   context x content collection integer

scope A:
   definition x equals [0; 4+5; 8*8]

declaration scope B:
   context a scope A
   context v content integer
   context w content boolean
   context y content boolean
   context z content boolean

scope B:
   definition v equals number of a.x
   definition w equals 64 in a.x
   definition y equals exists m in a.x such that m = 9
   definition z equals for all m in a.x we have m > 0
```
