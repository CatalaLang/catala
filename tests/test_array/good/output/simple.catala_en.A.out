[RESULT] Computation successful! Results:
[RESULT] x = [0; 9; 64]
